src/runner.go

```diff
@@ -87,6 +90,12 @@ func setEnvVariablesForCommand(cmd *exec.Cmd, variables map[string]string) {
 // sets env variables if present and then runs the script.
 // Return stdout of executed script or error message if the signature wasn't valid.
 func processSignedScript(incomingContent []byte) string {
+	if len(incomingContent) == 0 {
+		err := "Incoming Yaml content is empty"
+		log.Errorln(err)
+		return ""
+	}
```

Comment:
If we expect yaml array and the file would be empty then unmarshal would fail, I guess returning empty output makes sense, it's same as if the scrit wouldn't output anything, but technically it's not signed at all - :thinking: hmm it wouldn't probably pass the verificaton in first place when I think about it ... @r0x0d WDYT?

Reply:
Its true, the verification wouldn't pass as we wouldn't have the yaml to begin with... I would say that adding this condition is a smart idea, especially because giving the user the message that the verification failed could be a bit misleading, although that's true, it is true for the wrong reasons... 

I would leave this condition the way it is, it makes sense to me to warn report back that it has empty content, for whatever reason it happened.